test/include/pcl/test/pcl_macros.h

```diff
@@ -0,0 +1,62 @@
+/*
+ * Software License Agreement (BSD License)
+ *
+ *  Point Cloud Library (PCL) - www.pointclouds.org
+ *  Copyright (c) 2010-2012, Willow Garage, Inc.
```

Comment:
Copyright is 2019 onward in this situation and it it attributed to Open Perception. Here's an up-to-date example
https://github.com/PointCloudLibrary/pcl/blob/f5997d30ac7a4827c9b70895fdd5cb0f39df872f/common/include/pcl/make_shared.h#L5